src/cmd/src/cli/export.rs

```diff
@@ -417,3 +420,76 @@ fn split_database(database: &str) -> Result<(String, Option<String>)> {
         Ok((catalog.to_string(), Some(schema.to_string())))
     }
 }
+
+#[cfg(all(test, target_os = "linux"))]
+mod tests {
+    use std::path::PathBuf;
+    use std::process::{Command, Stdio};
+    use std::time::Duration;
+
+    use client::{Client, Database, DEFAULT_CATALOG_NAME, DEFAULT_SCHEMA_NAME};
+
+    #[tokio::test(flavor = "multi_thread")]
+    async fn test_export_create_table_with_quoted_names() {
+        let mut path = PathBuf::from(std::env::var("CARGO_MANIFEST_DIR").unwrap());
+        path.push("../../target/debug");
+
+        let mut standalone = Command::new("./greptime")
+            .current_dir(&path)
+            .args(["standalone", "start"])
+            .stdout(Stdio::null())
+            .spawn()
+            .unwrap();
+        // wait for fully start
+        std::thread::sleep(Duration::from_secs(3));
+
+        let client = Client::with_urls(["127.0.0.1:4001"]);
+        let database = Database::new(DEFAULT_CATALOG_NAME, DEFAULT_SCHEMA_NAME, client);
+        let sql = r#"
+            CREATE DATABASE "cli.export.create_table";
+        "#;
+        let res = database.sql(sql).await;
+        assert!(res.is_ok());
+
+        let sql = r#"
+            CREATE TABLE "cli.export.create_table"."a.b.c"(
+                ts TIMESTAMP,
+                TIME INDEX (ts)
+            ) engine=mito;
```

Comment:
Create database with `complex` database name to avoid conflict with existing database, maybe check `if not exists`? Same for following `create table`.